defaults/main.yml

```diff
@@ -370,12 +370,12 @@ security_sshd_disable_gssapi_auth: yes                       # V-204598
 # Disallow forwarding IPv4/IPv6 source routed packets on all interfaces
 # immediately and by default on new interfaces.
 security_disallow_source_routed_packet_forward_ipv4: yes     # V-72283 / V-72285
-security_disallow_source_routed_packet_forward_ipv6: yes     # V-72319
+security_disallow_source_routed_packet_forward_ipv6: no     # V-72319
 # Disallow responses to IPv4 ICMP echoes sent to broadcast address.
 security_disallow_echoes_broadcast_address: yes              # V-72287
 # Disallow IPV4 ICMP redirects on all interfaces immediately and by default on
 # new interfaces.
-security_disallow_icmp_redirects: yes                        # V-73175 / V-72289 / V-72291 / V-72293
```

Comment:
Is this specific to ipv6 ?

Reply:
this opt out of changing what the current system has in terms of the icmp redirect, is not ipv6 specific

Reply:
meaning that if icmp redirect on a specific server is 1, it will not force change it to 0